src/main/java/io/phasetwo/service/auth/OrgAddUserAuthenticatorFactory.java

```diff
@@ -61,6 +62,10 @@ private void addUser(AuthenticationFlowContext context) {
             org.getName(), context.getUser().getUsername());
         org.grantMembership(context.getUser());
         // TODO default roles from config??
+        context.getEvent()
+                .user(context.getUser())
+                .detail("joinedOrganization", org.getId())
```

Comment:
Keys are `snake_case` in details. Change to `joined_organization`

Reply:
Done